.clusterfuzzlite/build.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash -eu
+fuzzer=fuzz_loadCertificates
+cd "test/fuzz/pem"
+
+go-fuzz -tags go-fuzz -func FuzzLoadCertificates -o $fuzzer  .
+
+
+$CXX $CXXFLAGS $LIB_FUZZING_ENGINE -v $fuzzer -o $fuzzer.a
```

Comment:
One other reason why this might not be working is this needs to write the binary to `$OUT` (i.e. https://github.com/google/oss-fuzz/pull/6890/files#diff-cba4fa737b6c0f1b05e2ae53ffe49fbf946cc9d6094abe918d572f50178260b8) but again `compile_go_fuzzer` should take care of this.

Reply:
Thanks @oliverchang , I took the change that you had helped with oss-fuzz and used it here.